src/chains/ethereum/options/src/miner-options.ts

```diff
@@ -123,6 +123,7 @@ export type MinerConfig = {
       rawType: string | number;
       type: Address | number;
       hasDefault: true;
+      cliType: string | number;
```

Comment:
A though: Yargs supports multiple types, but only if you don't specify a type... which makes me wonder if it will treat `--option true` as a string or a boolean.

Also, agreed! This type is redundant.